evm/src/cpu/columns/general.rs

```diff
@@ -91,5 +113,67 @@ pub(crate) struct CpuLogicView<T: Copy> {
     pub(crate) output: [T; 16],
 }
 
+#[derive(Copy, Clone)]
+pub(crate) struct CpuJumpsView<T: Copy> {
+    // Assuming a limb size of 32 bits.
+    // The top stack value at entry (for jumps, the address; for `EXIT_KERNEL`, the address and new
```

Comment:
nit: I think `///` is nice for comments like this, even though it's not public. so tools know to associate it with the field. With IntelliJ it enables some nice hover text.

Reply:
Sure, I’ll change it!